includes/wikia/services/WikiService.class.php

```diff
@@ -817,18 +817,25 @@ public function getDetails( Array $wikiIds = null ) {
 			$rows = $db->select(
 				array(
 					'city_visualization',
-					'city_list'
+					'city_list',
+					'city_verticals',
+					'city_cat_mapping',
+					'city_cats'
 				),
 				array(
 					'city_list.city_id',
 					'city_list.city_title',
+					'city_list.city_dbname',
+					'city_list.city_description',
 					'city_list.city_url',
 					'city_list.city_lang',
-					'city_visualization.city_vertical',
+					'city_list.city_vertical',
 					'city_visualization.city_headline',
 					'city_visualization.city_description',
 					'city_visualization.city_main_image',
 					'city_visualization.city_flags',
+					'city_verticals.vertical_name',
+					'city_cats.cat_name'
 				),
 				array(
 					'city_list.city_public' => 1,
```

Comment:
```php
'city_list.city_id IN (' . implode( ',', $wikiIds ) . ')',
```

can be rewritten to prevent SQL injection:

```php
'city_list.city_id' => $wikiIds
```